gef.py

```diff
@@ -2808,6 +2819,14 @@ class X86(Architecture):
     _ptrsize = 4
     _endianness = Endianness.LITTLE_ENDIAN
 
+    # TODO: How do I show that this is a GefMemoryMapProvider?
+    @staticmethod
+    def x():
+        yield from itertools.chain(GefMemoryManager.parse_procfs_maps(),
+                                   # GefMemoryManager.parse_gdb_info_sections(),
```

Comment:
This would parse it from the ELF, which is nice, but we would get duplicate sections since we also get valid sections from procfs. Maybe we should have a way to resolve these? try it with this uncommented and then run `vmmap`